docs/source/en/using-diffusers/sdxl_turbo.md

```diff
@@ -49,11 +49,13 @@ pipeline = StableDiffusionXLPipeline.from_single_file(
 pipeline = pipeline.to("cuda")
 ```
 
+In this method, you need to set `timestep_spacing` parameter of the `StableDiffusionXLPipeline`'s scheduler to `'trailing'`.
```

Comment:
Can you modify the code snippet to show this?

Reply:
Still being discussed here: #6112. Let's wait for what they say.

Reply:
I think something like this should work no? Then also mention that users can play around with the scheduler config values, but the main one is to set `timestep_spacing="trailing"`.

```py
from diffusers import StableDiffusionXLPipeline, EulerAncestralDiscreteScheduler
import torch
pipeline = StableDiffusionXLPipeline.from_single_file(
    "https://huggingface.co/stabilityai/sdxl-turbo/blob/main/sd_xl_turbo_1.0_fp16.safetensors", torch_dtype=torch.float16)
pipeline.scheduler = EulerAncestralDiscreteScheduler.from_config(pipeline.scheduler.config, timestep_spacing="trailing")

prompt = "A croissant shaped like a cute bear."

image = pipeline(prompt=prompt, guidance_scale=0.0, num_inference_steps=1).images[0]
image
```